src/commands/swapNote.ts

```diff
@@ -17,7 +16,6 @@ const swapNote: Command = {
   },
   svg: ConvertToNoteIcon,
   exec: dispatch => {
-    asyncFocus()
```

Comment:
@raineorshine Presumably there was a reason why this was called in the first place. I found the change in cybersemics/em@1286df98513de8ba3be37cd3d4d152cf6a74406d but I wasn't able to find an associated PR or other comments on how this came about. Please let me know if there is other behavior that you want to see.

Reply:
Thanks for investigating. asyncFocus is used to open the virtual keyboard, so you're right that is not desired when it starts closed. 

Can you confirm that if the virtual keyboard is open at the beginning of the test case, it stays open through all steps? I want to make sure this fix doesn't cause a regression in editing mode.

Reply:
Yes, it does stay open. Drawing the gesture doesn't seem to affect the browser selection.